micro-rdk/src/native/entry.rs

```diff
@@ -69,13 +69,14 @@ pub async fn serve_web_inner(
                     }
                     Err(err) => {
                         if let Some(datetime) = cfg_received_datetime {
-                            let logs = vec![config_log_entry(datetime, Some(&err))];
+                            let logs = vec![config_log_entry(datetime, Some(err))];
                             client
                                 .push_logs(logs)
                                 .await
                                 .expect("could not push logs to app");
                         }
-                        panic!("{}", err)
+                        //TODO shouldn't panic here
+                        panic!("couldn't build robot");
```

Comment:
do we not want to print the error in the panic message?

Reply:
No it's more a a placeholder for when we will implement offline mode or try building again